components/email/src/Email.svelte

```diff
@@ -534,50 +534,54 @@
       message: message,
       type,
     });
+    const subject = message.subject?.toLowerCase().startsWith("re:")
+      ? message.subject
+      : `Re: ${message.subject}`;
+    const value = {
+      reply_to_message_id: message.id,
+      from,
+      to,
+      reply_to: from,
+      cc,
+      bcc: message.bcc,
+      body: message.body,
+      subject: subject,
+    };
+
+    let event_identifier;
+    switch (type) {
+      case "reply":
+        event_identifier = "replyClicked";
+        break;
+
+      case "reply_all":
+        event_identifier = "replyAllClicked";
+        break;
+    }
 
     //Check existing draft
     const existingDraft = activeThread?.drafts?.find(
       (draft) => draft.reply_to_message_id === message.id,
     );
     if (existingDraft) {
-      existingDraft.to = to;
-      existingDraft.cc = cc;
-      dispatchDraftEvent(event, existingDraft);
+      if (!_this.thread && existingDraft.id) {
+        const res = await fetchIndividualMessage(existingDraft.id);
+        existingDraft.body = res.body;
+        value.body = res.body;
+      }
+      dispatchEvent(event_identifier, {
+        event,
+        message: { ...existingDraft, subject, to, cc },
+        thread: activeThread,
+        value,
+      });
     } else {
       //Creating new reply message
-      const subject = message.subject?.toLowerCase().startsWith("re:")
-        ? message.subject
-        : `Re: ${message.subject}`;
-
-      let event_identifier;
-
-      switch (type) {
-        case "reply":
-          event_identifier = "replyClicked";
-          break;
-
-        case "reply_all":
-          event_identifier = "replyAllClicked";
-          break;
-      }
-
-      const value = {
-        reply_to_message_id: message.id,
-        from,
-        to,
-        reply_to: from,
-        cc,
-        bcc: message.bcc,
-        body: message.body,
-        subject: subject,
-      };
```

Comment:
@yifanplanet is this intended?

Reply:
@AaronDDM Yes, the code that defines the `subject`, `event_identifier` and `value` are move outside of the if condition.